WalletWasabi.Documentation/FAQ.md

```diff
@@ -0,0 +1,178 @@
+FAQ for [Wasabi Wallet](https://github.com/zkSNACKs/WalletWasabi)
+By [6102bitcoin](https://twitter.com/6102bitcoin)
+
+Note: Many of these Q&A have been copied from real users, see footer for acknowledgements. 
+
+# Index
+- Pre-Install
+- Install
+- Pre-Mix
+- Mixing
+- Post-Mix
+- Meta
+- Acknowledgements
+
+# Pre-Install
+
+### Who is behind Wasabi?
+
+The Company that is developing Wasabi is zkSNACKs LTD ([twitter](@Zksnacks_LTD) | [website](https://zksnacks.com/))  
+Members as of 2019.05.08:
+
+| Name              | Twitter                                                 | Position			   |
+| ------------------| --------------------------------------------------------|	-----------------------|
+| Gergely Hajdú 	| [@gergely_hajdu](https://twitter.com/gergely_hajdu) 	  | Co-CEO & Co-Founder	   |
+| Bálint Harmat		| [@bharmat84](https://twitter.com/bharmat84) 			  | Co-CEO & Co-Founder    |
+| Adam Ficsór		| [@nopara73](https://twitter.com/nopara73)	              | Developer & Co-Founder |
+| Lucas Ontivero	| [@lontivero](https://twitter.com/)                      | Developer & Co-Founder |
```

Comment:
@lontivero @bharmat @hgergoil It's ok if we call Lucas Co-Founder, right? He was here when we started the company.

Reply:
This is true but legally not correct.